src/parser/user-extractor-module.ts

```diff
@@ -42,19 +43,31 @@ export class UserExtractorModule implements Module {
     return sortedPriceLabels[0];
   }
 
+  _getAssigneeCount(issue: GitHubIssue) {
```

Comment:
```suggestion
  _getTaskMultiplier(issue: GitHubIssue) {
```
Count doesn't really reflect what this function returns

Reply:
Renamed accordingly: https://github.com/ubiquibot/conversation-rewards/pull/50/commits/9069d3f047ba053173de7589939319258e242981